### PR TITLE
Term Automation added

### LINKED ISSUE #266: add new term Automation
### What it is
Automation automates manual procedures and static scripts

### Problem it addresses
Helps administrators and operators manage environments by automating previously manual operations.

### How it helps
Speeds up administration and configuration of environments

### PR DESCRIPTION
**Pull request by mayank-cse**
Added the term **Automation**

### CODE REVIEW

**Comment by CathPag** [comment]
Thanks for your contribution, @mayank-cse! Could you please review the [Style Guide ](https://glossary.cncf.io/style-guide/) and make sure your definition is consistent and written for the same audience? Thank you!

**Comment by mayank-cse** [comment]
Hi @CathPag,
As you suggested I have formatted the definition as per the style guide, and have removed the types of automation thus making it easier to understand for the non-technical audience.
Kindly review the same and suggest changes if required 
thank you :)

**Comment by seokho-son** [comment]
Hello @mayank-cse :) 
Thank you for the contribution. Could you change (edit) the title of this PR? 
A concise and meaningful title will help collaborators :)

**Comment by mayank-cse** [comment]
Oh yeah sorry

**Comment by mayank-cse** [comment]
Hi,
@seokho-son 
Can you check it now

**Comment by mayank-cse** [comment]
Hi sir/ma'am,
@CathPag @seokho-son @jonasbn @caniszczyk 
Any thought or suggestion are welcome😄

**Comment by jonasbn** [comment]
I believe you mean @JasonMorgan

**Comment by mayank-cse** [comment]
yeah sorry sir
But if you also have any suggestions for the PR to improve, will be great if you could share

File: content/en/automation.md
Comment (JasonMorgan) [inline content/en/automation.md:10]: Good start, lets expand on this section.

File: content/en/automation.md
Comment (JasonMorgan) [inline content/en/automation.md:14]: Need a lot more here. The focus should be describing the problem in more depth.

File: content/en/automation.md
Comment (JasonMorgan) [inline content/en/automation.md:18]: I think concepts omit the various sections and just describe the thing. Look at Abstraction as an example. Regardless, we'll need to avoid any gendered language. Swap "man power" for something more neutral like, "engineering time".

**Review by JasonMorgan — CHANGES_REQUESTED** [review]

File: content/en/automation.md
Comment (mayank-cse) [reply content/en/automation.md]: Sir considering your suggestion, I have made few changes 
I have also tried explaining the term, with the help of example
Kindly check and suggest more, if anything else is required 
thank you :)

File: content/en/automation.md
Comment (mayank-cse) [reply content/en/automation.md:15]: Sir considering your suggestion, I have made few changes 
I have also tried explaining the term, with the help of example
Kindly check and suggest more, if anything else is required 
thank you :)

**Comment by mayank-cse** [comment]
Hi @JasonMorgan Sir @CathPag ma'am considering your suggestions, I have made few changes
I have also tried explaining the term, with the help of example
Kindly check and suggest more, if anything else is required
thank you :)

**Comment by mayank-cse** [comment]
Hi sir/ma'am,
@CathPag @seokho-son @JasonMorgan  @caniszczyk
Any thought or suggestion are welcome😄

**Comment by CathPag** [comment]
@mayank-cse, please allow us a little more time to review. We have one maintainer on vacation, the rest of the team is busy with work, and there are other contributors waiting for feedback as well. We'll get back as soon as we can. Thank you for your patience!

**Comment by mayank-cse** [comment]
Really sorry @CathPag ma'am
I had no intention to bother you guy's, I was just making sure if you are even getting notified and won't disagree was even excited 😅 as this is my first contribution to such a big organization like CNCF

*You guy's can take your time and help me whenever you are free
Will be waiting to learn 😄 from you guy's
Thank you

**Comment by CathPag** [comment]
No worries. People tend to forget that this is just a hobby for us too :)

**Comment by mayank-cse** [comment]
Yeah, no problem ma'am take your time
I'll be waiting for your inputs

thank you :)

**Comment by JasonMorgan** [comment]
I think this is a good start to automation and if you're willing to leave it as "Feedback Appreciated" we can merge it in. I don't believe it's ready to publish but it's much further along than it was. @mayank-cse please let me know what you think.

**Comment by mayank-cse** [comment]
Yes Sir @JasonMorgan ,

If it could be merged with feedback appreciated It's fine for me, it will also provide opportunities to the new brains to work over the definition

Thank You😄

**Comment by mayank-cse** [comment]
Hi @CathPag  ma'am, 
kindly suggest something to push this PR forward

thank you ma'am

**Review by CathPag — APPROVED** [review]
LGTM for "feedback appreciated"

**Review by JasonMorgan — APPROVED** [review]
Great work!

**Comment by JasonMorgan** [comment]
Hey @mayank-cse this one is ready to merge once you get the spellchecker happy with the PR. Thanks so much for your work and your patience!

**Comment by mayank-cse** [comment]
Hi @JasonMorgan Sir,
I am really sorry for the mistakes I am doing from starting, this being one of my first contributions I am learning a lot from you guys.
Sir I have tried removing the spelling errors, whenever you get time kindly approve the workflow.

Thank you for your support :)

**Comment by JasonMorgan** [comment]
@mayank-cse thanks for the great work! It seems the spellchecker is still angry about "chatbot" you'll need to update the wordlist to get this merged. Feel free to reach out to me on the CNCF slack if you need help.

**Comment by mayank-cse** [comment]
Thank You @JasonMorgan Sir! 
Your words really support me :)

Sir, how can I update the wordlist to get this merged 
Kindly guide me

**Pull request merged**
